docs/development/releasing.rst

```diff
@@ -1,233 +1,386 @@
+   if you have just branched ``v5.0.x``, create the ``v5.1.dev`` tag::
 
-   To find the statistics and contributors, use the `generate_releaserst.xsh`_
-   script. This requires `xonsh <https://xon.sh/>`_ and `docopt
-   <http://docopt.org/>`_ which you can install with::
+      $ git tag -s "v<next_version>.dev" -m "Back to development: v<next_version>"
 
-      pip install xonsh docopt
+#. Push all of these changes up to github::
 
-   You should then run the script in the root of the astropy repository as follows::
+      $ git push upstream v<version>.x:v<version>.x
+      $ git push upstream main:main
 
-      xonsh generate_releaserst.xsh 4.3 v5.0.dev \
-                                    --project-name=astropy \
-                                    --pretty-project-name=astropy \
-                                    --pat=<a GitHub personal access token>
+#. On the GitHub issue tracker, add a new milestone for the next major version
+   and for the next bugfix version, and also create a ``backport-v<version>.x``
+   label which can be used to label pull requests that should be backported
+   to the new release branch. You can then start to move any issues and pull
+   requests that you know will not be included in the release to the next milestones.
 
-   The first argument should be the last major version (before any bug fix
-   releases, while the second argument should be the ``.dev`` tag that was just
-   after the branching of the last major version. Finally, you will need a
-   GitHub personal access token with default permissions (no scopes selected).
+#. Inform the Astropy developer community that the branching has occurred.
 
-   The output will look similar to::
+.. _release-procedure-first-rc:
 
-      This release of astropy contains 2573 commits in 163 merged pull requests
-      closing 104 issues from 98 people, 50 of which are first-time contributors
-      to astropy.
+Releasing the first major release candidate
+===========================================
 
-      * 2573 commits have been added since 4.3
-      * 104 issues have been closed since 4.3
-      * 163 pull requests have been merged since 4.3
-      * 98 people have contributed since 4.3
-      * 50 of which are new contributors
+.. _release-procedure-update-iers:
 
-      The people who have contributed to the code for this release are:
+Updating the IERS parameter and leap second tables
+--------------------------------------------------
 
-      - Name 1 *
-      - Name 2 *
-      - Name 3
+Ensure the built-in IERS earth rotation parameter and leap second tables are up
+to date by changing directory to ``astropy/utils/iers/data`` and executing
+``update_builtin_iers.sh``. Check the result with ``git diff`` (do not be
+surprised to find many lines in the ``eopc04_IAU2000.62-now`` file change; those
+data are reanalyzed periodically) and committing. Since in some cases updating
+the IERS tables may result in test failures, this update should be done via a
```

Comment:
not just the test failures, but the principle, too: development happens on the main branch, and it doesn't hurt the dev branch to have updated numbers, too.

Reply:
Done